main.go

```diff
@@ -230,10 +215,9 @@ func main() {
 	}
 
 	// Only cache if npm command is install, node_modules could be included in the repository
-	if (len(args) != 0) && (args[0] == "install") {
-		err := cacheNpm(workdir, parseCacheLevel(config.CacheLevel))
-		if err != nil {
-			log.Warnf("Falied to mark files for caching, error: %s", err)
+	if config.UseCache && (len(args) != 0) && (args[0] == "install" || args[0] == "i" || args[0] == "add") {
```

Comment:
this is still unreliable, do we need to check if this flag is given or not? simply if the user wants to cache, and there is a node modules dir then just cache it, isn't it?

Reply:
node_modules could be commited to the repository, or added by a previous step.